### ISSUE TITLE
Feature request: Option for database password prompt for showing passwords in clear text

### ISSUE THREAD

**Issue by aeisi** [keepassxreboot/keepassxc#2222]
This is an alternative to #2223:

It would be beneficial to have the option to show passwords in clear text only after entering the database password.
I am using keepassxc to store my passwords and access them via the browser extension because I do not want to use the built in password manager of the browser. That is the reason why I would like to leave my keepassxc database unlocked so that the browser extension can access the credentials.
If I forget to lock my desktop it would be possible for others to see my passwords in keepassxc. Therefore an option to reenter the database password before showing my passwords would be desirable.

## Expected Behavior
An option to require the database password when showing the password in clear text with unlocked databases.

## Current Behavior
There is no such option.

**Comment by droidmonkey** [comment]
Duplicate of #1809
